x-pack/plugins/uptime/server/lib/adapters/pings/elasticsearch_pings_adapter.ts

```diff
@@ -78,6 +79,7 @@ export class ElasticsearchPingsAdapter implements UMPingsAdapter {
     }
     const params = {
       index: INDEX_NAMES.HEARTBEAT,
+      rest_total_hits_as_int: true,
```

Comment:
I could be wrong, but I think this option is not needed here.

Reply:
I think we *do* need this. It was causing a failure, but I can double-check.

Reply:
Yeah. Here's the reason we need it: `const total: number = get(bucket, 'bucket_total.value');` It's not needed in Kibana 7, but in Kibana 6.7, we need it so that we force es6.7 and es7.0 to give us consistent response payloads.

Reply:
Hm. You may be right. I was mixing this up with the previous query, I think. Good catch!